extension/src/extension.ts

```diff
@@ -191,9 +193,17 @@ export class Extension extends Disposable {
     this.dispose.track(
       commands.registerCommand(RegisteredCommands.STOP_EXPERIMENT, async () => {
         const stopWatch = new StopWatch()
-        const wasRunning = this.dvcRunner.isExperimentRunning()
+        const dvcLiveOnlyPids = await this.experiments.getDvcLiveOnlyPids()
+        const wasRunning =
+          this.dvcRunner.isExperimentRunning() ||
+          definedAndNonEmpty(dvcLiveOnlyPids)
         try {
-          const stopped = await this.dvcRunner.stop()
+          const [dvcLiveOnlyStopped, runnerStopped] = await Promise.all([
+            stopProcesses(dvcLiveOnlyPids),
+            this.dvcRunner.stop()
```

Comment:
[I] We can only stop experiments under these circumstances. Need to set another context value and hide `dvc.stopRunningExperiment` if we cannot stop the process.

Reply:
added